src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV40.sql

```diff
@@ -0,0 +1,272 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     DeleteInstanceV40
+--
+-- FIRST SCHEMA VERSION
+--     6
+--
+-- DESCRIPTION
+--     Removes the specified instance(s) and places them in the DeletedInstance table for later removal
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The Partition key
+--     @cleanupAfter
+--         * The date time offset that the instance can be cleaned up.
+--     @createdStatus
+--         * Status value representing the created state.
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.DeleteInstanceV40
```

Comment:
I don't think this is changing the signature for the deletion sproc, so you can instead update the existing one